### ISSUE TITLE
ofAppGlutWindow mouse y offset ubuntu, compile error osx

### ISSUE THREAD

**Issue by thomasgeissl** [openframeworks/openFrameworks#4322]
I am trying to run this little program:

```c++
#include "ofMain.h"
#include "ofAppGlutWindow.h"

class ofApp : public ofBaseApp{
        void draw()
        {
                ofDrawCircle(x,y, 25);
        }

        void mousePressed(int x, int y, int button)
        {
                this->x = x;
                this->y = y;
        }
        float x,y;
};
int main(  int argc, char *argv[]  ){
        ofAppGlutWindow window;
        ofSetupOpenGL(&window, 1024,768, OF_WINDOW);
        ofRunApp( new ofApp());
}
```
The pointer position should set the center of the circle. But there is an offset on the y axis. It is about 25 pixels. Could be the height of the window header. If I resize the window, it works fine.

![bug-glut-mouse-offset](https://cloud.githubusercontent.com/assets/3796674/9570089/e21f4f94-4f7f-11e5-8cb5-40fa82afc637.jpg)

If I try to compile it on osx 10.10 using the makefiles I get the following error:

```make
Undefined symbols for architecture x86_64:
  "_glutDragEventFunc", referenced from:
      ofAppGlutWindow::setup(ofGLWindowSettings const&) in libopenFrameworks.a(ofAppGlutWindow.o)
```

Here is the initial issue: https://github.com/openframeworks/projectGenerator/issues/16

**Comment by arturoc** [comment]
do you need glut for any specific reason? we can look into this issues but glut is kind of deprecated and only there for compatibility with old applications

**Comment by thomasgeissl** [comment]
I just realised it when I used projectGeneratorLegacy, the linux version still uses glut. 
And I was using it for an old project which depends on libraries already using glut. But I can modify it to work with glfw. That should not be a problem.

**Comment by thomasgeissl** [comment]
It works normally on debian on a different machine. I think it is not worth to investigate this issue, at least not the linux part. It probably just happens on my laptop and I do not think it is an of bug.

**Comment by arturoc** [comment]
if i remember correctly this is a bug in some version of unity, i think glfw has a workaround for it. let's close it then

**Comment by arturoc** [comment]
i've also changed the PG legacy to not use glut any more

**Comment by thomasgeissl** [comment]
Okay, perfect. Thanks.
Is it tricky to fix the os x compilation error? I think this should be fixed for 0.9.0.
I might have some time this week to look into this.

**Comment by skyl** [comment]
I'm on OSX and I've been getting this here and there. I don't think I've gotten it with an official example. But, I get it now when I run, for example, https://github.com/patriciogonzalezvivo/ofxFX/tree/master/example-filters

```
Undefined symbols for architecture x86_64:
  "_glutDragEventFunc", referenced from:
      ofAppGlutWindow::setup(ofGLWindowSettings const&) in libopenFrameworks.a(ofAppGlutWindow.o)
ld: symbol(s) not found for architecture x86_64
```
